cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +33,11 @@ else()
   set(configure_command "./config")
 endif()
 
+# Pass fPIC through
+if(";${CMAKE_CXX_FLAGS};" MATCHES ";-fPIC;")
```

Comment:
> I do not think it is safe to send all CMAKE_CXX_FLAGS because ./config uses its own inbuilt flags that could conflict, such as -d.

If user specify such flags then we should use them. Global flags should came from toolchain and applied to all projects. The issue you have reported is exactly about it. Also it make sense to pass CMAKE_CXX_FLAGS_RELEASE and think about multiple-config installation of OpenSSL :)

> I chose -fPIC because it was causing me issues

I see nothing special about `-fPIC`, I expect same issue with `-fsanize=*` flags for instance.